src/main/kotlin/second_pass/signature/SignableMember.kt

```diff
@@ -0,0 +1,7 @@
+package second_pass.signature
+
+interface SignableMember {
+    fun getName(): String
```

Comment:
I can't wrap my head around this set of classes related to the signature.

Do we really need so many, or is it enough to offer something along the lines of https://github.com/Clean-CaDET/platform/blob/master/RepositoryCompiler/CodeModel/CaDETModel/CodeItems/CaDETMember.cs#L20-L37